conda/install.py

```diff
@@ -1244,17 +1228,18 @@ def main():
 
     prefix = opts.prefix
     pkgs_dir = join(prefix, 'pkgs')
-    pkgs_dirs[0] = [pkgs_dir]
+    pkgs_dirs = [pkgs_dir]
```

Comment:
It turns out this change was wrong, and the original is correct. The reason is that `pkgs_dirs` is a global. So `pkgs_dirs[0]` accesses the global variable, but `pkgs_dirs =` creates a new local instead. _This_ is why the package cache is empty: because it never sees the proper value of the package cache directory.

Alas, that's not the only problem.